fv3core/utils/corners.py

```diff
@@ -111,14 +111,14 @@ def fill_corners_2cells_mult_x(
     return q
 
 
-def fill_corners_2cells_x_stencil(q: FloatField):
+def fill_corners_2cells_x_stencil(q: FloatField, q_corner: FloatField):
```

Comment:
I can't really tell what `q_corner` means even if I read the code without going into the fill_corners_... function, is this the `q` that gets read from in order to write into `q`'s corners? If so can you involve `_out` and `_in` in the names somehow or make this clear in some other way?